examples/pytorch/CaptumExample/Titanic_Captum_Interpret.py

```diff
@@ -0,0 +1,323 @@
+    ax1.hist(cond_vals[:, 9], 100)
+    ax1.set(title="Neuron 9 Distribution")
+    ax2.hist(cond_vals[:, 7], 100)
+    ax2.set(title="Neuron 7 Distribution")
+    mlflow.log_figure(fig, "Neurons_Distribution.png")
+
+
+def neuron_conductance(test_input_tensor, neuron_selector=None):
+    """
+    We have identified that some of the neurons are not learning important features, while others are. Can we now understand what each of these important neurons are looking at in the input? For instance, are they identifying different features in the input or similar ones?
+
+    To answer these questions, we can apply the third type of attributions available in Captum, **Neuron Attributions**. This allows us to understand what parts of the input contribute to activating a particular input neuron. For this example, we will apply Neuron Conductance, which divides the neuron's total conductance value into the contribution from each individual input feature.
+
+    To use Neuron Conductance, we create a NeuronConductance object, analogously to Conductance, passing in the model as well as the module (layer) whose output we would like to understand, in this case, net.sigmoid1, as before.
+    """
+    neuron_selector = 0
+    neuron_cond = NeuronConductance(net, net.sigmoid1)
+
+    # We can now obtain the neuron conductance values for all the test examples by calling attribute on the NeuronConductance object. Neuron Conductance requires the neuron index in the target layer for which attributions are requested as well as the target index for networks with mutliple outputs, similar to layer conductance. As before, we provide target = 1, corresponding to survival, and compute neuron conductance for neurons 0 and 10, the significant neurons identified above. The neuron index can be provided either as a tuple or as just an integer if the layer output is 1-dimensional.
+
+    neuron_cond_vals = neuron_cond.attribute(
+        test_input_tensor, neuron_selector=neuron_selector, target=1
+    )
+    neuron_cond, _ = visualize_importances(
+        feature_names,
+        neuron_cond_vals.mean(dim=0).detach().numpy(),
+        title="Average Feature Importances for Neuron {}".format(neuron_selector),
+    )
+    mlflow.log_text(
+        str(neuron_cond), "Avg_Feature_Importances_Neuron_" + str(neuron_selector) + ".txt"
+    )
+
+
+if __name__ == "__main__":
+
+    parser = ArgumentParser(description="Titanic Captum Example")
+
+    parser.add_argument(
+        "--use_pretrained_model",
+        default=False,
+        metavar="N",
+        help="Use pretrained model or train from the scratch",
+    )
+
+    parser.add_argument(
+        "--max_epochs",
+        type=int,
+        default=100,
+        metavar="N",
+        help="Number of epochs to be used for training",
+    )
+
+    parser.add_argument(
+        "--lr", type=float, default=0.1, metavar="LR", help="learning rate (default: 0.1)",
+    )
+
+    args = parser.parse_args()
+    dict_args = vars(args)
+
```

Comment:
To reflect MLflow best practices & keep the logic here clean, could we start our MLflow run here instead of [at the top-level of this module](https://github.com/mlflow/mlflow/pull/3866/files#diff-630321e96fc2d84abbac3715fc2d306f8136a7b6027707237f6f82a31af96ca8R20)? e.g.

```
with mlflow.start_run():
    (net, train_features, train_labels, test_features, test_labels, feature_names,) = train()
    train_input_tensor = train_step(train_features)
    test_input_tensor = test_step(test_features)
    feature_conductance(test_input_tensor)
    layer_conductance(test_input_tensor)
    neuron_conductance(test_input_tensor)
    mlflow.log_param("Train Size", len(train_labels))
    mlflow.log_param("Test Size", len(test_labels))

Reply:
@smurching Thanks for pointing it out. I was going to do that actually. I will update that as well